script.js

```diff
@@ -1,9 +1,20 @@
-const showFunction = (id) => {
-  console.log("clicked");
-  var password = document.getElementById(id);
-  if (password.type === "password") {
-    password.type = "text";
+const showCreatePassword = document.querySelector("#show-create-password");
+const showConfirmPassword = document.querySelector("#show-confirm-password");
+const createPassword = document.querySelector("#create-password");
+const confirmPassword = document.querySelector("#confirm-password");
+
+showCreatePassword.addEventListener("click", () => {
+  if (createPassword.type === "password") {
+    createPassword.type = "text";
   } else {
-    password.type = "password";
+    createPassword.type = "password";
   }
-};
+});
+
+showConfirmPassword.addEventListener("click", () => {
+  if (confirmPassword.type === "password") {
+    confirmPassword.type = "text";
+  } else {
+    confirmPassword.type = "password";
+  }
+});
```

Comment:
great work on the JS file its easily understandable

Reply:
Thank you